esp/esp/program/modules/handlers/creditcardmodule_stripe.py

```diff
@@ -316,6 +316,7 @@ def charge_payment(self, request, tl, one, two, module, extra, prog):
         if 'error_type' in context:
             #   If we got any sort of error, send an email to the admins and render an error page.
             self.send_error_email(request, context)
+            context['support_email'] = settings.DEFAULT_EMAIL_ADDRESSES['support']
```

Comment:
since we use render_to_response(), the context should already include the settings